app/src/main/java/com/android/brewr/ui/authentication/SignIn.kt

```diff
@@ -64,6 +65,15 @@ fun SignInScreen(userViewModel: UserViewModel, navigationActions: NavigationActi
   val launcher =
       rememberLauncherForActivityResult(ActivityResultContracts.StartActivityForResult()) { result
         ->
+        if (!isConnectedToInternet(context)) {
+          Toast.makeText(
```

Comment:
Is it possible to test this ? like a UI test ? Just worrying our line coverage...